types/oauth2orize/index.d.ts

```diff
@@ -3,17 +3,9 @@
 
 import { IncomingMessage, ServerResponse } from "http";
 
-declare global {
-    namespace Express {
-        interface Request {
-            oauth2?: OAuth2 | undefined;
-        }
-    }
-}
-
```

Comment:
i am not familiar with these types and someone more familiar with them needs to comment and see if this removal is ok. It seems not backward compatible.

Reply:
That's a fair point. I totally forgot that I removed this.
It has to be added by yourself in the project if you need the typing for the oauth2 property on Express Requests.
The issue is that I cannot override OAuth2<_any_, _any_> if it is already defined here.
I don't know if there is another solution to solve this.